docs/samples/custom/torchserve/logger/README.md

```diff
@@ -0,0 +1,133 @@
+# Inference Logger Demo
+
+First let us create a message dumper KNative service which will print out the Cloud Events it receives.
+We will use the following resource yaml:
+
+```yaml
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: message-dumper
+spec:
+  template:
+    spec:
+      containers:
+      - image: gcr.io/knative-releases/knative.dev/eventing-contrib/cmd/event_display
+```
+
+Let's apply the resource to the cluster:
+
+```bash
+kubectl create -f message-dumper.yaml
+```
+
+We can now create a torchserve predictor with a logger which points at the message dumper. The yaml is shown below.
+
+```yaml
+apiVersion: "serving.kubeflow.org/v1beta1"
+kind: "InferenceService"
+metadata:
+  name: "torchserve-logger"
+spec:
+  predictor:
+    minReplicas: 1
+    logger:
+      url: http://message-dumper.default/
+      mode: all
+    containers:
+      - image: {username}/torchserve:latest
+        name: torchserve-container
+```
+
+* By default the logger port is 8081 and component port is 8080. Run torchserve inference on 8080 and management on a port other than 8081.
```

Comment:
I have fixed this now, it uses the uncommon port https://github.com/kubeflow/kfserving/blob/master/cmd/agent/main.go#L37

Reply:
@yuzisun is it possible to have it configurable as torchserve with kfserving wrapper will not use 8080 port for inference.